docs/topics/crud-experience-intro.adoc

```diff
@@ -0,0 +1,24 @@
+The {ProductShortName} CRUD application Booster expands on the HTTP API Booster to provide a basic example of performing _create_, _read_, _update_ and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API.
```

Comment:
@cescoffier please take a look at this section and let me know if it looks correct